spec/plans/prepare.fmf

```diff
@@ -106,6 +108,10 @@ example: |
         supported, ``crb`` and ``fips`` are coming in the near
         future.
 
+        .. note::
+            Subpackage tmt+ansible need to be installed for this
+            feature to work.
+
```

Comment:
> Subpackage tmt+ansible need to be installed for this feature to work.

No, it doesn't :) What the plugin need is a working Ansible, which *may* be installed by installing `tmt+ansible`.

Also, these notes belong to plugin documentation, i.e. plugin docstrings, they are not part of the specification. Somewhere around here, I guess, https://github.com/teemtee/tmt/blob/main/tmt/steps/prepare/ansible.py#L79 ? And the same for `prepare/feature`.

How about this?

```
.. note::

   The plugin requires a working Ansible to be available, which can be achieved
   by installing ``tmt+ansible`` package.
```